### PR TITLE
Fixed drop down on top right icon

### PR DESCRIPTION
**Pull request by mla04762**
https://github.com/gordon-cs/gordon-360-ui/issues/1112

### CODE REVIEW

File: src/components/Header/components/NavButtonsRightCorner/index.jsx [hunk 1]
```diff
@@ -80,7 +80,7 @@ const GordonNavButtonsRightCorner = ({ onClose, openDialogBox, open, anchorEl })
     isAuthenticated && isSiteAdmin ? (
       <GordonNavButton
         unavailable={!isOnline ? 'offline' : null}
-        onLinkClick={onClose}
+        zonLinkClick={onClose}
```
Comment (bennettforkner) [inline src/components/Header/components/NavButtonsRightCorner/index.jsx:83]: Is this a mistype? I can't find anything referencing that property :)

Comment (mla04762) [reply src/components/Header/components/NavButtonsRightCorner/index.jsx]: I just reverted the error code and re-committed it.

**Comment by bennettforkner** [comment]
Hi @mla04762 -- Can you add a picture to the description above of how the menu looks after your changes? It's helpful for the review process for those who cannot look over your shoulder :)

**Comment by bennettforkner** [comment]
Awesome, LGTM! I'll let your team officially approve though.

FYI, images in the description of this PR might be better than in the issue. Although doing both doesn't hurt!

**Review by russtuck — APPROVED** [review]
LGTM

**Pull request merged**